playbooks/roles/jenkins/templates/jenkins_defaults.j2

```diff
@@ -11,7 +11,7 @@ JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8
 JAVA_ARGS="-Djava.awt.headless=true -Dfile.encoding=UTF8 -Xmx4096m -XX:MaxPermSize=1024m"  # Allow graphs etc. to work even when an X server is present
 
 # Allow JS and CSS on HTML Publisher pages
-JAVA_ARGS="${JAVA_ARGS} -Dhudson.model.DirectoryBrowserSupport.CSP=\"sandbox allow-scripts; default-src 'self' 'unsafe-inline' data:;\""
+JAVA_ARGS="${JAVA_ARGS} -Dhudson.model.DirectoryBrowserSupport.CSP=\"sandbox allow-scripts; default-src 'self' 'unsafe-inline' 'unsafe-eval' data:;\""
```

Comment:
Does this mean we'll have a new warning in the Jenkins UI to dismiss? 😹

Reply:
I don't think so? Jenkins is currently running with this setting and I haven't seen a warning.

Reply:
That's because they only show up when we restart Jenkins (and we then dismiss the warnings...).

Reply:
Right, of course. I can apply this and see whether it will cause an error? I'm not sure what we could do if it did though. Alternatively we can rollback the changes to BackstopJS until they remove the `eval` from their code, but I don't think that's currently on their roadmap.

Reply:
I think it's fine for now. Long term we might be thinking about alternatives to BackstopJS anyway...